IP_POLICY_GUIDANCE.md

```diff
@@ -0,0 +1,71 @@
+# OpenJS Foundation IP Policy Guidance
+
+The purpose of this guide is to make it straightforward for projects
+to follow the [OpenJS Foundation IP Policy][IP Policy].
+
+Of course, the source of truth remains the [IP Policy][].
+
+If in doubt about anything related to IP after reading this document,
+first read the [IP Policy][], and then [ask for help](#getting-help).
+
+## Basic requirements
+
+There are three key requirements for all OpenJS Foundation projects:
+
+1. [Licensing](#1-licensing),
+2. [copyright notices](#2-copyright-notices), and
+3. displaying the [standard website footer](#3-standard-website-footer) on the project's website.
+
+Additionally, projects may decide to adopt the [OpenJS Foundation Contributor License Agreement (CLA)](#contributor-license-agreement-cla).
+
+### 1. Licensing
+
+Code may be licensed under:
+  * [Apache, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+  * [MIT](https://opensource.org/licenses/MIT)
+  * [2-Clause BSD](https://opensource.org/licenses/BSD-2-Clause)
+  * [3-Clause BSD](https://opensource.org/licenses/BSD-3-Clause)
+
+Documentation may be licensed under:
+  * [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/)
+  * [MIT](https://opensource.org/licenses/MIT)
+
+If you need to use a license which isn't in the list,
+you will need to obtain permission from the Board to do so.
+
+### 2. Copyright notices
+
+The recommendation is to use a general copyright statement
+of the following form (where XYZ is the project's name):
+
+- **Copyright The XYZ Authors.**
+- **Copyright The XYZ Contributors.**
+- **Copyright Contributors to the XYZ project.**
+
+By using this format, the project avoids having to deal with
+names of copyright holders, years or ranges of years,
+and variations on the (c) symbol.
+
+_Please note that you **must not** change or remove existing copyright lines
+unless you put them there **and** have the right to do so.
+If there are existing copyright lines just
+add the recommended copyright statement below the existing copyright lines.
+You may however update copyright lines that say
```

Comment:
must?

Reply:
Yes, I'd recommend keeping this as "must." While these lines are informational in nature, they were likely put there for a specific purpose and a number of licenses require these to be distributed intact (unless, of course, the person who holds the copyright removes the line themselves).

Reply:
That comment isn't clear to me.

@boneskull are you suggesting the following change?

```suggestion
You must however update copyright lines that say
```

@brianwarner is that what you understood?

If that's the proposed change, I'd be opposed: this can create a lot of work for certain projects for no additional benefits but consistency.

Reply:
@tobie my read of @brianwarner's comment at least is that it applies to `_Please note that you **must not** change or remove existing copyright lines`, not sure about @boneskull's comment.  Unless its a hard requirement to update the copyright lines to reflect the OpenJS foundation I'd agree that `may` in that context makes the most sense.

Reply:
Sorry that was unclear, I was referring back to the last use of "must". 

Agreed, "may" is best here. The idea is that you can update any lines where you have the right, but you don't have to do so. At best, these lines are hints as to copyright ownership, and it would be asking projects to do busy-work to update them.

Reply:
@boneskull are you OK with not changing this?

Reply:
Pinging @boneskull, here.

Reply:
yes

Reply:
Awesome. Thank you.